zebra/debug_nl.c

```diff
@@ -576,6 +582,38 @@ const char *nhm_rta2str(int type)
 	}
 }
 
+static const char *ncm_rta2str(int type)
+{
+	switch (type) {
+	case NETCONFA_UNSPEC:
+		return "UNSPEC";
+	case NETCONFA_IFINDEX:
+		return "IFINDEX";
+	case NETCONFA_FORWARDING:
+		return "FORWARDING";
+	case NETCONFA_RP_FILTER:
+		return "RP_FILTER";
+	case NETCONFA_MC_FORWARDING:
+		return "MCAST";
+	case NETCONFA_PROXY_NEIGH:
+		return "PROXY_NEIGH";
+	case NETCONFA_IGNORE_ROUTES_WITH_LINKDOWN:
+		return "IGNORE_LINKDOWN";
+	case NETCONFA_INPUT:
+		return "MPLS";
+	case NETCONFA_BC_FORWARDING:
+		return "BC_FORWARDING";
```

Comment:
BCAST? Or it's not treated the same way as NETCONFA_MC_FORWARDING?

Reply:
sure- changed to BCAST